modules/administration-guide/partials/proc_using-chectl-to-install-che-on-codeready-containers.adoc

```diff
@@ -15,18 +15,18 @@ This section describes how to install {prod-short} on CodeReady Containers (CRC)
 
 .Procedure
 
-. Initiate the CRC platform:
-
-.. Configure your host machine for CodeReady Containers:
+.. Remove any previous cluster:
 +
 ----
-$ crc setup
+$ crc delete
 ----
 
-.. Remove any previous cluster:
+. Initiate the CRC platform:
+
+.. Configure your host machine for CodeReady Containers:
 +
 ----
-$ crc delete
+$ crc setup
 ----
 
 .. Initiate the `oc` command line interface within CRC: 
```

Comment:
```suggestion
. Initiate the `oc` command line interface within CRC: 
```

Reply:
This is really more like "allowing access to" than "initiating".

Reply:
```adoc
. Enable access to the `oc` command line interface embedded in CodeReady Containers:
```

Reply:
```suggestion
.. Enable access to the `oc` command line interface embedded in CodeReady Containers:
```